pysat/_orbits.py

```diff
@@ -383,22 +390,30 @@ def _getBasicOrbit(self, orbit=None):
 
             # ensure user is requesting a particular orbit
             if orbit is not None:
+                # set up data access for both pandas and xarray
+                self.sat.data = self._fullDayData#.copy()
```

Comment:
What's the benefit of leaving .copy() commented out here?

Reply:
Bad habits die hard.